cpp/CMakeLists.txt

```diff
@@ -212,17 +208,15 @@ endif()
 # ##################################################################################################
 # * cuvs ---------------------------------------------------------------------
 if(NOT BUILD_CPU_ONLY)
-  if(INSTALL_CPP_HEADERS)
-    add_library(cuvs_cpp_headers INTERFACE)
-    add_library(cuvs::cuvs_cpp_headers ALIAS cuvs_cpp_headers)
-    target_include_directories(
-      cuvs_cpp_headers
-      INTERFACE "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-                "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>"
-                "$<INSTALL_INTERFACE:include>"
-    )
-    target_link_libraries(cuvs_cpp_headers INTERFACE raft::raft rmm::rmm)
```

Comment:
Why did you remove these links? We leak rmm and raft from cuvs headers.

Reply:
An oversight since each target that consumes `cuvs_cpp_headers` also links to raft and rmm as well.